workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,20 +51,35 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
+        {
+            GetClientLogFilename();
+            Launch("Launch standalone client", "local worker launch UnityClient default", "*unityclient.log");
+        }
+
+        public static void LaunchLocalDeployment()
         {
             BuildConfig();
+            Launch("Launch SpatialOS locally", "local launch", "spatial_*.log");
+        }
 
-            var command = Common.SpatialBinary;
-            var commandArgs = "local launch";
-            
-            if (Application.platform == RuntimePlatform.OSXEditor)
-            {
-                command = "osascript";
-                commandArgs = $"-e 'tell application \"Terminal\"\nactivate\ndo script \"cd {SpatialProjectRootDir} && {Common.SpatialBinary} local launch\"\nend tell'";
-            }
+        private static string GetClientLogFilename()
+        {
+            string clientConfigFilename = "spatialos.UnityClient.worker.json";
+
+            var logConfigPath = Path.Combine(SpatialProjectRootDir, "workers", "unity", clientConfigFilename);
+            var configFileJson = File.ReadAllText(logConfigPath);
+            var dict = Json.Deserialize(configFileJson) as Dictionary<string, object>;
 
-            var processInfo = new ProcessStartInfo(command, commandArgs)
+            return "WIP";
+        }
+
+        public static void Launch(string commandName, string commandArgs, string logFileName)
```

Comment:
Passing strings around everywhere for launching different processes makes me feel uneasy. I'd much rather we just make these separate functions.

Reply:
Migrated to enums as recommended by @jamiebrynes7 and some dicts to mappings.